scylla/src/statement/prepared_statement.rs

```diff
@@ -610,4 +614,59 @@ mod tests {
             ],
         );
     }
+
+    struct ValuesMaker(SerializedValues);
+    impl ValuesMaker {
+        fn new() -> Self {
+            Self(SerializedValues::new())
+        }
+        fn value(mut self, v: impl Value) -> Self {
+            self.0.add_value(&v).unwrap();
+            self
+        }
+        fn finish(self) -> SerializedValues {
+            self.0
+        }
+    }
+
+    #[derive(Debug)]
+    struct PtrEq<'t, T>(&'t T);
+    impl<'t, T> PartialEq for PtrEq<'t, T> {
+        fn eq(&self, other: &Self) -> bool {
+            std::ptr::eq(self.0, other.0)
+        }
+    }
+    impl<'t, T> Eq for PtrEq<'t, T> {}
```

Comment:
Do we need this? As far as I see, it's used in conjunction with `&ColumnSpec`, and for those you have derived the `Eq` trait.

Reply:
I though the same. However, `test` attribute (as in `cfg(test)`) does not propagate through crate boundary. That is, if I compile tests for `scylla` crate, `PartialEq` is not implemented for structs living in `scylla_cql`.

Reply:
Then let's not overcomplicate. `ColumnSpec` is a part of the API so we can just derive `Eq` and `PartialEq` for it always.